app/dashboard/src/components/AriaComponents/Inputs/Input/Input.tsx

```diff
@@ -73,10 +74,13 @@ export const Input = forwardRef(function Input<
     variant,
     variants = INPUT_STYLES,
     fieldVariants,
-    form,
+    form: formRaw,
     autoFocus = false,
     ...inputProps
   } = props
+  const formFromContext = Form.useFormContext()
```

Comment:
we can pass form prop in context and context will return the passed form if it's defined, otherwise will try to consume a form from the context.